File: src/sdk-reference/php/3/collection/delete-specifications/snippets/delete-specifications-1.php

<?php

use \Kuzzle\Kuzzle;

$kuzzle = new Kuzzle('localhost');
$dataCollection = $kuzzle->collection('collection', 'index');

// Deleting one document
try {
  $result = $dataCollection->deleteSpecifications();
}
catch (ErrorException $e) {

}

File: .repos/sdk/sdk-android-3/doc/3/core-classes/collection/delete-specifications/snippets/delete-specifications-1.java

// Deleting one document
kuzzle
  .collection("collection", "index")
  .deleteSpecifications(new ResponseListener<JSONObject>() {
    @Override
    public void onSuccess(JSONObject result) {

    }

    @Override
    public void onError(JSONObject error) {
      // Handle error
    }
  });
